src/Microsoft.AspNetCore.DataProtection/RC1ForwardingActivatorCollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.DataProtection.Internal;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.DataProtection
+{
+    public static class RC1ForwardingActivatorCollectionExtensions
+    {
+        public static IDataProtectionBuilder AddRC1KeySupport(this IDataProtectionBuilder builder)
```

Comment:
Any reason we couldn't just make this work by default rather than requiring people specify `AddRC1KeySupport`?